test/routes/src/server/routes/t_launchpad.js

```diff
@@ -944,6 +944,58 @@ describe('The Launchpad API endpoint', () => {
                 done(err);
               });
           });
+
+          context('when snaps are memcached', () => {
+            const urlPrefix = 'https://github.com/anowner/';
+            const repositoryUrl = 'https://github.com/anowner/aname';
+            let snap;
+
+            before(() => {
```

Comment:
Maybe `beforeEach` and `afterEach` here to avoid confusion in case somebody wants to add another test to this `context` block.